File: week-02/day-02/ex-variables/cuboid.py
cub_height = 12
cub_length = 33
cub_width = 20

surface = ((cub_height * cub_length) + (cub_length * cub_width) + (cub_height * cub_width)) * 2
volume = cub_height * cub_length * cub_width

print("Surface area: " + str(surface))
print("volume: " + str(volume))

File: week-07/day-02/cuboid.js
'use strict';

// Write a program that stores 3 sides of a cuboid as variables (floats)
// The program should write the surface area and volume of the cuboid like:

var height = 12;
var length = 33;
var width = 20;

var surface = 2 * ((height * length) + (length * width) + (width * height))
var volume = height * length * width

console.log('the surface is ' + surface)
console.log('the volume is ' + volume)
